newIDE/app/src/ProjectCreation/CreateProjectDialog.js

```diff
@@ -56,6 +57,22 @@ export const createNewProject = async (): Promise<?NewProjectSource> => {
   };
 };
 
+export const createNewProjectWithDefaultLogin = async (): Promise<?NewProjectSource> => {
+  const url =
+    'https://resources.gdevelop.io/examples-database/login-template.json';
```

Comment:
Ah that's a very interesting idea!

Reply:
Should this be in a subfolder if this is coming from GDevelop-examples? Or is this something added manually?